projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -405,8 +408,10 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
     /**
      * @hidden @internal
      */
-    public onToggleClosing(e: CancelableBrowserEventArgs) {
-        this.onClosing.emit(e);
+    public onToggleClosing(e: CancelableBrowserEventArgs & IBaseEventArgs) {
+        const eventArgs: CancelableBrowserEventArgs & IBaseEventArgs = Object.assign({}, e, { owner: this });
+        this.onClosing.emit(eventArgs);
+        e.cancel = eventArgs.cancel;
```

Comment:
it is not great optimization, but if `e.cancel === true` we can return here and not set `_scrollPosition`.

Reply:
That's _pretty safe_, I'll add it, thanks!